mapper/mapper.go

```diff
@@ -123,21 +123,30 @@ func CreateSelector(req *http.Request, basePage coreModel.Page, dimName, lang st
 }
 
 // CreateAreaTypeSelector maps data to the Overview model
-func CreateAreaTypeSelector(req *http.Request, basePage coreModel.Page, lang string, areaType []dimension.AreaType, selectionName string) model.Selector {
+func CreateAreaTypeSelector(req *http.Request, basePage coreModel.Page, lang string, areaType []dimension.AreaType, selectionName string, isValidationError bool) model.Selector {
 	p := CreateSelector(req, basePage, selectionName, lang)
 	p.Page.Metadata.Title = "Area Type"
 
+	if isValidationError {
+		p.Page.Error = coreModel.Error{
+			Title: "SelectAreaTypeError",
```

Comment:
This needs to be a meaningful string that should start with 'Error'. Changes are needed to the `dp-renderer` library to return a localisation.
```suggestion
			Title: "Error: Select an area type",
```